awswrangler/distributed/ray/modin/_data_types.py

```diff
@@ -13,7 +13,7 @@ def pyarrow_types_from_pandas_distributed(
     df: pd.DataFrame, index: bool, ignore_cols: Optional[List[str]] = None, index_left: bool = False
 ) -> Dict[str, pa.DataType]:
     """Extract the related Pyarrow data types from a pandas DataFrame."""
-    func = ray_remote(pyarrow_types_from_pandas)
+    func = ray_remote(scheduling_strategy="DEFAULT")(pyarrow_types_from_pandas)
```

Comment:
Isn't "DEFAULT" already the default anyways? If not perhaps it can be set with our `ray_remote` definition instead of repeating it across the codebase?

Reply:
Yes actually I initially set it to `PACK` which optimises for data locality but looks like it's only available for placement groups but not as a default remote option for some reason. I've opened an issue with ray to clarify this. Will remove it for now, thanks.